packages/node-webpack-startup-snapshot-checker/src/index.spec.ts

```diff
@@ -1,25 +1,52 @@
-/* eslint-disable no-console */
-import { execFile as execFileCallback } from 'child_process';
-import { promisify } from 'util';
+import { spawn } from 'cross-spawn';
 import path from 'path';
 import { expect } from 'chai';
-const execFile = promisify(execFileCallback);
 
 describe('startup snapshot checker', function () {
+  const execFile = (
+    command: string,
+    args: ReadonlyArray<string>
+  ): Promise<void> => {
+    const proc = spawn(command, args);
```

Comment:
Using `execFile` from `child_process` was causing `ENOENT` trying to run `npx` on windows.